demos/button/icons.html

```diff
@@ -11,37 +11,56 @@
 	<link rel="stylesheet" href="../demos.css">
 	<script>
 	$(function() {
-		$( "button:first" ).button({
-			icons: {
-				primary: "ui-icon-locked"
-			},
-			text: false
+		$( ".widget button, .widget a" ).first().button({
+			icon: "ui-icon-gear",
+			showLabel: false
 		}).next().button({
-			icons: {
-				primary: "ui-icon-locked"
-			}
+			icon: "ui-icon-gear"
 		}).next().button({
-			icons: {
-				primary: "ui-icon-gear",
-				secondary: "ui-icon-triangle-1-s"
-			}
+			icon: "ui-icon-gear",
+			iconPosition: "end"
 		}).next().button({
-			icons: {
-				primary: "ui-icon-gear",
-				secondary: "ui-icon-triangle-1-s"
-			},
-			text: false
+			icon: "ui-icon-gear",
+			iconPosition: "bottom"
+		}).next().button({
+			icon: "ui-icon-gear",
+			iconPosition: "top"
 		});
 	});
 	</script>
 </head>
 <body>
+<div class="widget">
+	<h1>Widget</h1>
+	<a href="#">Button with icon only</a>
+	<button>Button with icon on the left</button>
+	<button>Button with icon on the right</button>
+	<button>Button with icon on the bottom</button>
+	<button>Button with icon on the top</button>
+</div>
+<div class="css">
+	<h1>CSS</h1>
+	<button class="ui-button ui-widget ui-corner-all ui-button-icon-only" title="Button with icon only">
+		<span class="ui-icon ui-icon-gear"></span>Button with icon only
+	</button>
+	<button class="ui-button ui-widget ui-corner-all ui-icon-beginning">
+		Button with no text
+	</button>
+	<button class="ui-button ui-widget ui-corner-all ui-icon-beginning">
+		<span class="ui-icon ui-icon-triangle-1-w"></span>Button with icon on the left
+	</button>
+	<button class="ui-button ui-widget ui-corner-all ui-icon-end">
+		Button with icon on the right<span class="ui-icon ui-icon-triangle-1-e"></span>
+	</button>
+	<button class="ui-button ui-widget ui-corner-all">
+		Button with icon on bottom<span class="ui-icon ui-icon-triangle-1-s ui-icon-display-block"></span>
```

Comment:
This class literally just adds `display:block` to the element and centers it. Open to other suggestions.